tests/subscript1.py

```diff
@@ -0,0 +1,5 @@
+def test_subscript():
+    s: str
+    s = 'abc'
+    s = s[0]
+    s = s[1:2]
```

Comment:
I have to check how strings are handled. Let's test this on arrays, in addition to str:
```
A: i32[5]
B: i32[2]
i: i32
i = A[0]
B = A[1:3]
```
Or something like that.